app/views/bookmarks/list.html.erb

```diff
@@ -100,7 +102,7 @@
             <%= '-' %>
           <% end %>
         </td>
-        <!--Edit/Destroy Bookmarks-->
+        <!--Editing/destroying bookmarks is available to the reviewing user, the reviewing user's team, and the instructor.-->
```

Comment:
"Editing/destroying bookmarks is available to the user who added the bookmark, that user's team, and the instructor."